docs/storage/storage.md

```diff
@@ -19,10 +19,27 @@ limitations under the License.
 -->
 ### Notebook Storage
 
-In Zeppelin there are two option for storage Notebook, by default the notebook is storage in the notebook folder in your local File System and the second option is S3.
+Zeppelin a pluggable notebook storage mechanism with multiple implementations.
+There are few Notebook storage options avaialble for a use,
```

Comment:
Is it possible to have multiple storage simultaneously? Could we clarify in this doc?